x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/data.ts

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
This can be replaced by adding a new file called `epm.ts` to [`use_request/`](https://github.com/elastic/kibana/tree/feature-ingest/x-pack/plugins/ingest_manager/public/applications/ingest_manager/hooks/use_request) that looks a lot like the `agent_config.ts` file in that directory